skimage/color/colorconv.py

```diff
@@ -824,8 +824,12 @@ def gray2rgb(image, alpha=None):
     Raises
     ------
     ValueError
-        If the input is not a 2- or 3-dimensional image.
+        If the input is not a 1-, 2- or 3-dimensional image.
 
+    Notes
+    -----
+    If the input is a 1-dimensional image of shape ``(M, )``, the output
+    will be shape ``(M, 1, 3)``.
```

Comment:
@jni no, there were no discussions on this point. Your proposal is the most reasonable one. +1.
(I was just thinking that many of the functions in skimage don't support multichannel 1D, so it is better to upcast the shape)